packages/nedit/SPECS/nedit.spec

```diff
@@ -79,6 +79,7 @@ desktop-file-install \
 
 %files
 %doc README ReleaseNotes
+/usr/lib/desktop/iconcatalog/pages/C/RSE/*
```

Comment:
This is the only change that we need to make to the specs for inclusion in the icon catalog. I don't know how we feel about the RPMs managing stuff outside of our prefix, _but_ uninstalling the RPM does also take the icon catalog entry with it by doing this!!